.gitignore

```diff
@@ -1,10 +1,10 @@
 .DS_Store
 node_modules
 .project
-dist/
```

Comment:
Yup - that was added with `build-gh-pages` and shouldn't be used any longer. If you have a `dist/` folder left in your repo, you can just trash it.